howtos/configuration.md

```diff
@@ -10,9 +10,26 @@ The documentation can be customized by setting parameters in [docusaurus.config.
 
 ## Search by Algolia
 
-- API-Key and Index name is configured in `./docusaurus.config.js`.
+Internal search is handled by an integration with Algolia DocSearch.
 
-[Algolia DocSearch](https://docsearch.algolia.com/) configuration is **NOT** a part of this repository. Config can be found on a dedicated [Docsearch Configurations](https://github.com/algolia/docsearch-configs) repo owned by Algolia but maintained by the individual projects. You must prove you are associated with the project to modify the configuration in the [Camunda config](https://github.com/algolia/docsearch-configs/blob/master/configs/camunda.json).
+### Configuration
+
+Our Algolia API key, app ID, and index name are configured in `./docusaurus.config.js`.
+
+Configuration of Algolia's index and crawler can be edited at https://crawler.algolia.com/.
```

Comment:
Configuration of Algolia's index and crawler can be edited with the [Algolia Crawler](https://crawler.algolia.com/), or something of the like?

Reply:
```suggestion
Configuration of Algolia's index and crawler can be edited with the [Algolia Crawler](https://crawler.algolia.com/).
```